sdk/tables/Azure.Data.Tables/src/TableEntity.cs

```diff
@@ -58,8 +58,8 @@ public string RowKey
         /// <value>An <see cref="ETag"/> containing the ETag value for the entity.</value>
         public ETag ETag
         {
-            get { return new ETag(GetString(TableConstants.PropertyNames.EtagOdata)); }
-            set { _properties[TableConstants.PropertyNames.EtagOdata] = value.ToString(); }
+            get { return new ETag(GetString(TableConstants.PropertyNames.ETag)); }
+            set { _properties[TableConstants.PropertyNames.ETag] = value.ToString(); }
```

Comment:
Safe to assume that ETag is a struct and cannot be `null` here?

Reply:
correct